spec/index.bs

```diff
@@ -7575,31 +7578,47 @@ GPUQueue includes GPUObjectBase;
 
             **Returns:** {{undefined}}
 
-            - If |source|.{{GPUImageCopyExternalImage/source}}
-                <l spec=html>[=is not origin-clean=]</l>,
+            1. Let |sourceImage| be |source|.{{GPUImageCopyExternalImage/source}}
+            1. Run [=Check the usability of the image argument=](|sourceImage|).
+                If it throws an exception, stop.
+                If it does not return `good`, throw an {{InvalidStateException}} and stop.
```

Comment:
I'm not sure this will cover the case below: 
During the CopyExternalImageToTexture/ImportToTexture call, it might possible we hit some browser internal error which cause the externalImage content is  invalid. 
If not, then I think we need to define a behaviour on this case.
Fail silently is one option. (Not good, but webgl and canvas impl in chromium works like this)

Reply:
Honestly I don't know what we are supposed to do here. Ideally specs shouldn't need to say what to do if some _unspecified_ problem occurred inside the browser. Hopefully those unspecified problems fall into the following buckets:

- Resource was lost due to gpu process loss. In this case the device is lost, too so it doesn't matter.
- Resource is in an "invalid" state because it hasn't been initialized or is in some other state that means, per-spec, it would be a blank image. In those cases we make sure the implementation copies from a blank image.